content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -219,57 +234,63 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
 
-We highly recommend using `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` to set `env`, `service`, and `version` for your services.
-Check out the [Unified Service Tagging][7] documentation for recommendations on how to configure these environment variables.
+### Automatic instrumentation optional configuration
 
-#### Instrumentation
+The following table lists configuration variables that are available **only** when using automatic instrumentation.
+
+| Setting Name                                                    |Description                                                                                                                                                                                                     |
+| --------------------------------------------------------------  | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `DD_TRACE_ENABLED`<br/><br/>`TraceEnabled`                      | Enables or disables all automatic instrumentation. Setting the environment variable to `false` completely disables the CLR profiler. For other configuration methods, the CLR profiler is still loaded, but traces will not be generated. Valid values are: `true` (default) or `false`. |
+| `DD_TRACE_LOG_DIRECTORY`                                        | Sets the directory for .NET Tracer logs.<br/><br/>Default: `%ProgramData%\Datadog .NET Tracer\logs\`                                                                                                                                                                                     |
+| `DD_TRACE_LOG_PATH`                                             | Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set.                                                                                                                    |
+| `DD_DISABLED_INTEGRATIONS`<br/><br/>`DisabledIntegrationNames`  | Sets a list of integrations to disable. All other integrations remain enabled. If not set, all integrations are enabled. Supports multiple values separated with semicolons. Valid values are the integration names listed in the [Integrations][8] section.                             |
+| `DD_TRACE_ADONET_EXCLUDED_TYPES`<br/><br/>`AdoNetExcludedTypes` | Sets a list of `AdoNet` types (for example, `System.Data.SqlClient.SqlCommand`) that will be excluded from automatic instrumentation. |
+
+### Additional optional configuration
+
+The following table lists the supported configuration variables that are available for both automatic (`.MSI`) and custom (`NuGet`) instrumentation.
```

Comment:
It seems unnecessary to have the "(`.MSI`)" and "(`NuGet`)" in the sentence. I'd recommend removing them